app/scripts/controllers/network/provider-api-tests/shared-tests.js

```diff
@@ -0,0 +1,708 @@
+/* eslint-disable jest/require-top-level-describe, jest/no-export, jest/no-identical-title */
+
+import { fill } from 'lodash';
+import {
+  withMockedInfuraCommunications,
+  withInfuraClient,
+  buildMockParamsWithoutBlockParamAt,
+  buildMockParamsWithBlockParamAt,
+  buildRequestWithReplacedBlockParam,
+} from './helpers';
+
+export function testsForRpcMethodNotHandledByMiddleware(
```

Comment:
Nit: it might be better to call this something like `testsForRpcMethodNotCachedByMiddleware`. It's still _handled_, just not cached.

Reply:
Also worth having a test to ensure it's not cached. Like the "identical requests" tests used elsewhere, or the "pending" block tag test.

Reply:
>it might be better to call this something like

Actually... on second thought maybe the original name is better. The middleware does more than caching.